### ISSUE TITLE
Fuel consumption is unrealistic

### ISSUE THREAD

**Issue by dsatgh** [legoboyvdlp/A320-family#334]
I noticed unrealistic fuel consumption while using a flight plan with fuel calculations. After a search I found the following:
[https://en.wikipedia.org/wiki/CFM_International_CFM56](url)
[https://en.wikipedia.org/wiki/CFM_International_LEAP](url)
In the CFM56 link there are two TSFC's, one for take off and one for cruise. It seems the values in the cfm56-5b4_1.xml and cfm56-5b4_2.xml are the take off values. When I changed them to the cruise values the fuel consumption was close to what I expected. The same for the other engine options. I couldn't find the values for the V2527-A5 and PW1127G-JM engines, but for a first guess they could be set the same as the CFM engines.

To sum it up, there should be the following TSFC changes in the jsbsim configurations:
cfm56-5b4 and v2527-a5:     0.335 -> 0.545
leap-1a26 and pw1127g:      0.328 -> 0.510

**Comment by Octal450** [comment]
Unfortunately TSFC in JSBsim does not equal real life quoted numbers. You can't just plug in the numbers in my experience.

I assume fuel consumption is too low at cruise for N1 settings? I'll have to check this.

Kind Regards,
Josh

**Comment by dsatgh** [comment]
Yes it's too low. Thanks for looking into this.

Kind Regards,
Daniel

**Comment by Octal450** [comment]
I see the problem. I'll have to recalculate. May take some days, busy with college starting up.

**Comment by Octal450** [comment]
I'm accepting the changes for now but I'll have to go through and fine tune it later.

I'd request if you can test it please vs SimBrief and such.

Josh

**Comment by dsatgh** [comment]
Thanks Josh. I did a 2h flight with the A320-214 and compared it to a real world data sample. It's already so close that I could not with certainty tell what to improve more, because there are so many factors influencing the fuel consumption.

**Comment by dsatgh** [comment]
I did some additional tests. From takeoff to landing the fuel consumption is now spot on and the taxi/idle fuel consumption is double of what it should be.

**Comment by Octal450** [comment]
Nice. I did tweak it slightly. I'll leave it be now.

Re idle, I'd have to do significant work to fix that. Not sure about how to tackle at the moment.

Best,
Josh
